src/components/HomepageCards/HomepageCard.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+import Badge from "react-bootstrap/Badge";
+
+import { selectMode } from "../../store/user/selectors";
+
+import "./HomepageCards.css";
+
+export default function HomepageCard(props) {
+  const mode = useSelector(selectMode());
+  const colorScheme = mode ? ["#222", "#FFF"] : ["#FFF", "#222"];
```

Comment:
Second time i see this, not good practice to have it several times in your code, do a small helper function like
```
const getColorScheme = (mode) => mode ? ["#222", "#FFF"] : ["#FFF", "#222"];
// and call it from your components
const colorScheme = getColorScheme(mode)
```